.eleventy.js

```diff
@@ -1,17 +1,26 @@
+// TODO: After Canary 19:
+// const { EleventyHtmlBasePlugin } = require("@11ty/eleventy");
+
 module.exports = function (eleventyConfig) {
-  eleventyConfig.addPassthroughCopy("assets");
-  let pathPrefix = "/";
+  // TODO: After Canary 19:
+  // eleventyConfig.addPlugin(EleventyHtmlBasePlugin);
+  eleventyConfig.addPassthroughCopy("src/assets");
+	eleventyConfig.setBrowserSyncConfig({
```

Comment:
This is okay for now, but we'll probably have to update in the near future. 

11ty will remove in version 2.0. From their [docs](https://www.11ty.dev/docs/server-browsersync/).

>Starting with Eleventy 2.0, the [Eleventy Dev Server](https://www.11ty.dev/docs/dev-server/) is now the default stock development server in Eleventy core. This section of documentation only applies to Eleventy 1.x and 0.x and will be removed when Eleventy 2.0 is stable. If you want to use Browsersync with Eleventy 2.0, learn how to swap [back to Browsersync](https://www.11ty.dev/docs/dev-server/#swap-back-to-browsersync).

Reply:
**Sidenote:** It takes 4.7 seconds to compile USWDS SASS. We should investigate why it's taking so long and see how USWDS/Compile can be sped up.

Reply:
Yup, I want to update to the 11ty canary asap, but it currently has a bug with `pathPrefix` so we can't use it yet. One it does, we can drop browsersync AND use the new `EleventyHtmlBasePlugin` plugin